pkg/frontend/admin_openshiftcluster_vmresize_test.go

```diff
@@ -44,7 +43,7 @@ func TestAdminVMResize(t *testing.T) {
 	for _, tt := range []*test{
 		{
 			name:       "basic coverage",
-			vmName:     "aro-fake-node-0",
+			vmName:     "aro-fake-master-node-0",
```

Comment:
can we update the names of the test nodes to reflect how they are named in production?